htdocs/accountancy/class/accountancyexport.class.php

```diff
@@ -163,13 +163,13 @@ public function exportNormal($objectLines) {
 
 		foreach ( $objectLines as $line ) {
 			// Std export
-			$date = dol_print_date($line->doc_date, $conf->global->ACCOUNTING_EXPORT_DATE);
+			$date = dol_print_date($line->doc_date, '%d/%m/%Y');
 			print $date . $this->separator;
 			print $line->doc_ref . $this->separator;
 			print length_accountg($line->numero_compte) . $this->separator;
 			print length_accounta($line->code_tiers) . $this->separator;
-			print price($line->debit) . $this->separator;
-			print price($line->credit) . $this->separator;
+			print '"' . price($line->debit) . '"' . $this->separator;
```

Comment:
when you export a price of 396,96€ will give you 2 columns one with 396 the other with 96